modules/ROOT/pages/index.adoc

```diff
@@ -78,6 +78,33 @@ include::{partialsdir}maintenance-release-schedule-link.adoc[]
 These are the older ownCloud releases.
 Users of these releases are *strongly encouraged* to upgrade to the latest production release.
 
+=== ownCloud 10.3
```

Comment:
Why do we link all the versions here at all? There is the version switcher on the left side...

Reply:
It was standard practice for the longest time, to link to outdated versions.

Reply:
Back in that time there haven't be any version selector like we got within the antora navigation.

Reply:
@pako81 @cdamken @voroyam should the links to older versions be removed?

Reply:
Honestly - I use the version switcher. I haven't seen this site since I was looking for an outdated version to test stuff, like oC 9.1.

I think we should only link the versions that are no longer in the version switcher.

Reply:
latest three versions have been removed from the previous releases list. Can this be approved now?

Reply:
IMHO we should have only the last 2 available (3 could be ok too), but master should not appear (not even in the search)

Reply:
that means "latest" should be the lastest with number and not master